packages/react/src/TreeView/TreeView.test.tsx

```diff
@@ -1392,6 +1393,10 @@ describe('State', () => {
 
 describe('Asynchronous loading', () => {
   it('updates aria live region when loading is done', () => {
+    // Set up live region for this test
+    const liveRegionEl = document.createElement('live-region')
+    document.body.appendChild(liveRegionEl)
```

Comment:
The live region utilities should be doing this automatically, let me know if not 👀 

I think test-wise we just have to have a clean up for them, e.g.

```tsx
afterEach(() => {
  const liveRegion = document.querySelector('live-region');
  document.body.removeChild(liveRegion);
});
```